integrated_channels/cornerstone/utils.py

```diff
@@ -53,23 +50,18 @@ def create_cornerstone_learner_data(request, course_id):
 
 def convert_invalid_course_id(course_id):
```

Comment:
after this PR, do we even use this function anymore?

Reply:
we do but i might be able to factor it out, here is one of the places we use it https://github.com/edx/edx-enterprise/blob/master/integrated_channels/cornerstone/exporters/content_metadata.py#L63